resources/views/v2/components/FrontpageSearchItem/FrontpageSearchItem.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+
+    <a :href="result.route" class="FrontpageSearchItem" :class="isclasses">
+        {{ result.title }}
+         <span :class="'FrontpageSearchItem__badge FrontpageSearchItem__badge--' + (result.badge_color ? result.badge_color : 'orange')" v-if="result.badge">
```

Comment:
- Do we need the badges at all?
- Why not we use isclasses for dynamic coloring? I know its awkward with vue->vue but this is what we have not. Lets declare colors later as JS so they can be used both from CSS and JS